flask_project/campaign_manager/insights_functions/feature_attribute_completeness.py

```diff
@@ -119,9 +119,14 @@ def check_feature_completeness(
                         warning_message.append(warning)
 
         feature_data['error'] = 'False'
+        feature_data['warning'] = 'False'
+
         if warning_message or error_message:
```

Comment:
We are still generating errors based on warnings here. Shouldn't this just check for error_messages?